docs/api/app.md

```diff
@@ -396,6 +396,15 @@ You can request the following paths by the name:
 * `videos` Directory for a user's videos.
 * `pepperFlashSystemPlugin`  Full path to the system version of the Pepper Flash plugin.
 
+### `app.getFileIcon(path, [options], callback)`
```

Comment:
I chatted about it with @deepak1556 and he suggested doing it under app namespace